tools/bottlerocket-variant/src/lib.rs

```diff
@@ -70,10 +70,10 @@ pub mod error {
 ///
 /// ```rust
 /// use bottlerocket_variant::{Variant, VARIANT_ENV};
-/// std::env::set_var(VARIANT_ENV, "metal-k8s-1.24");
+/// std::env::set_var(VARIANT_ENV, "vm-k8s-1.32");
```

Comment:
shouldn't this be vmware-k8s-1.32?
```suggestion
/// std::env::set_var(VARIANT_ENV, "vmware-k8s-1.32");
```

Reply:
Ah thanks for the catch, changing